series/plzen-pyvo/events/2021-09-08.yaml

```diff
@@ -0,0 +1,57 @@
+city: plzen
+start: 2021-09-08 18:00:00
+name: Plzeňské Pyvo
+number: 22
+topic: Python a vědecké a průmyslové výpočtys
+venue: hub-20
+description: |
+  Vážení přátelé a příznivci našich setkání!
+  
+  ...
+  
+  V HUBu bude k dispozici voda, točená Kofola, pivo a pár drobností k poškádlení chuťových buněk. Pokud máte zájem se na akci najíst, lze využít dovážky, kousek od objektu, v ulici Černická 10, je například provozovna Pizza doma. 
+  
+  Prosíme o dodržování rozumných opatření zabraňujících šíření nakažlivých nemocí. [Link](https://covid.gov.cz/situace/obchod-sluzby/restaurace-bary-kavarny)
+
+  Protože se nekteří mohou zúčastnit jen online, plánujem část našeho setkání streamovat, pokud ale můžete, raději přijděte na živo. 
+  
+  [Link](https://us02web.zoom.us/j/83241542579?pwd=MlVvV3l1cm9zYWVHaUtKN3FFSHNuZz09)
+  Meeting ID: 832 4154 2579
+  Passcode: 1407
+  
+  Děkujeme a na viděnou!
+
+talks:
+  - title: Úvodní slovo
+    speakers:
+      - Tomáš Bedřich & Filip Vaculík & Tomáš Zítka
+  - title: Python a náročné výpočty - jde to?
+    speakers:
+      - Tomáš Zítka
+    description: Acceleration to the rescue!
+  - title: Sfepy
+    speakers:
+      - Robert Cimrman, NTC
+    description: Představení balíku pro FEM výpočty postaveného na Numpy a C extensions.
+  - title: Tensorflow a CUDA
+    speakers:
+      - David Žahour
+    description: 
+  - title: PyTorch
+    speakers:
+      - ????
+    description: 
+    description: 
```

Comment:
```suggestion
```

Reply:
Srovnal jsem formátování a tohle opravil, snad by to teď mělo projít testy.